src/compute/src/render/errors.rs

```diff
@@ -67,16 +65,12 @@ impl<T, E> MaybeValidatingRow<T, E> for Result<T, E> {
 /// dataflow is in the process of shutting down.
 #[derive(Clone)]
 pub(super) struct ErrorLogger {
-    shutdown_probe: ShutdownProbe,
```

Comment:
Most of the doc comments on `ErrorLogger` and its methods are now stale because they talk about shutdown. Can you fix those? Also `log_always` doesn't make sense anymore.

We could consider removing `ErrorLogger`. The only thing it does now is do the `warn!`/`error!` logging pattern, which I think we agreed to get rid of anyway?

Reply:
Thanks, fixed. I didn't remove the error logger just yet because it'd change a bunch of call-sites, but it's a great call-out!